raster-foundry/raster-foundry
# Add feature flags for browsing external imagery sources

## Overview

This PR adds a migration that will add feature flags for both the external imagery source connection feature in general and the connection to the Planet API specifically.

Since our move to OIDC compliance, accessing Auth0 profile metadata is not possible. For this reason, this PR also changes feature-flag lookup to use the default values provided by the config object.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Notes

If only using server-side feature-flags ends up being sufficient, we'll want to go in and cleanup the feature-flag process a bit.


## Testing Instructions

 * Run migrations
 * Check that the external source select is not visible
 * Try changing the `active` values of the two feature-flags and see that they work as expected.

Closes #2865
